lib/phoenix_component.ex

```diff
@@ -150,12 +155,18 @@ defmodule Phoenix.Component do
         """
       end
 
-  And now you can invoke it as:
+  When calling the component, you can use the special attribute `:let` (note
+  the leading `:`) to take the value that the component passes back and bind it
+  to a variable:
 
-      <.unordered_list :let={entry} entries={~w(apple banana cherry)}>
-        I like <%= entry %>
+      <.unordered_list :let={an_entry} entries={~w(apple banana cherry)}>
```

Comment:
Maybe it was just me, but I needed to experiment to see for sure that `:let` determines what the variable name is, so making it slightly different here shows that this is the case.